spec/support/test_paths.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module TestPaths
+  extend self
+
+  APP_NAME = "dummy_app"
```

Comment:
I'd probably use another name, but this is for backward compatibility with current feature specs.